data/json/items/gun/380.json

```diff
@@ -60,5 +60,24 @@
     "dispersion": 480,
     "durability": 8,
     "magazines": [ [ "380", [ "rugerlcpmag" ] ] ]
+  },
+  {
+    "id": "hptcf380",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Hi-Point CF-380",
+    "description": "The Hi-Point CF-380 is a blowback operated semi automatic pistol designed by Hi-Point Firearms, which is known for making inexpensive firearms, and for making said firearms bulky, uncomfortable, and absolutely invincible.",
+    "weight": 709,
+    "volume": "167 ml",
+    "price": 7500,
+    "to_hit": -4,
+    "bashing": 8,
+    "material": [ "plastic", "steel" ],
```

Comment:
I'd add budget_steel (zinc) as a material here, and all the other hi-points.

Reply:
We have a zinc material, so I'd add that instead.